ravenframework/CodeInterfaces.py

```diff
@@ -1,53 +0,0 @@
-# Copyright 2017 Battelle Energy Alliance, LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""
-Created on April 14, 2014
-
-@author: alfoa
-
-comment: The CodeInterface Module is an Handler.
-         It inquires all the modules contained in the folder './CodeInterfaceClasses'
-         and load them, constructing a '__interFaceDict' on the fly
-"""
-#for future compatibility with Python 3--------------------------------------------------------------
-from __future__ import division, print_function, unicode_literals, absolute_import
-#End compatibility block for Python 3----------------------------------------------------------------
-
-#External Modules------------------------------------------------------------------------------------
-import os
-from glob import glob
-import inspect
-#External Modules End--------------------------------------------------------------------------------
-
-#Internal Modules------------------------------------------------------------------------------------
-from .EntityFactoryBase import EntityFactory
-from .utils import utils
-#Internal Modules End--------------------------------------------------------------------------------
-
-__moduleInterfaceList = []
-startDir = os.path.join(os.path.dirname(__file__),'CodeInterfaceClasses')
-for dirr,_,_ in os.walk(startDir):
-  __moduleInterfaceList.extend(glob(os.path.join(dirr,"*.py")))
-  utils.add_path(dirr)
```

Comment:
I completely approve of removing this line :)  It made the sys.path excessively long.